---
title: MSSQLSERVER_26013 | Документация Майкрософт
ms.custom: ''
ms.date: 03/06/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.technology: supportability
ms.topic: conceptual
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
author: MashaMSFT
ms.author: mathoma
ms.openlocfilehash: 9c1600212ae0d719a520a4af565c069960c890a0
ms.sourcegitcommit: ad4d92dce894592a259721a1571b1d8736abacdb
ms.translationtype: MT
ms.contentlocale: ru-RU
ms.lasthandoff: 08/04/2020
ms.locfileid: "87664259"
---
# <a name="mssqlserver_26013"></a>MSSQLSERVER_26013
    
## <a name="details"></a>Сведения  
  
|||  
|-|-|  
|Название продукта|SQL Server|  
|Идентификатор события|26013|  
|Источник события|MSSQLSERVER|  
|Компонент|SQLEngine|  
|Символическое имя|SNI_SSL_READY|  
|Текст сообщения|Сертификат [Cert Hash(sha1) "%hs"] был успешно загружен для шифрования.|  
  
## <a name="explanation"></a>Объяснение  
 Это информационное сообщение. Указанный сертификат был успешно загружен в [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] и может использоваться для шифрования.  
  
## <a name="user-action"></a>Действие пользователя  
 Никаких действий не требуется.  
  
  
